Makefile.am

```diff
@@ -225,3 +225,28 @@ EXTRA_DIST += \
   config/stl_hash.m4 \
   mac_build.sh \
   ola.spec
+
+# -----------------------------------------------------------------------------
+
+# Create a linter target
+.PHONY : lint
+lint: flake8 cpplint
+
+# flake8 linter
+.PHONY : flake8
+flake8: Makefile.am
+if FOUND_FLAKE8
+	flake8
+else
+	$(error flake8 not found. Install flake8 and re-run configure.)
+endif
+
+# cpplint linter
+CPP_LINT_URL = "https://raw.githubusercontent.com/google/styleguide/gh-pages/cpplint/cpplint.py"
+.PHONY : cpplint
+cpplint: Makefile.am cpplint.py
```

Comment:
Actually you could do best of both here and use the FOUND_CPPLINT or install it if missing!

Reply:
Good idea! I pushed some changes a bit ago to address this.

Reply:
Great thanks.